File: Taller Tipos de clases/JAVA/Clase Normal/Persona.java
class Persona {
    String nombre;
    int edad;

    // Constructor sin parámetros
    public Persona() {
        this.nombre = "Sin nombre";
        this.edad = 0;
    }

    // Constructor con parámetros
    public Persona(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    // Constructor sobrecargado
    public Persona(String nombre) {
        this.nombre = nombre;
        this.edad = 18;
    }

    // Constructor copia
    public Persona(Persona p) {
        this.nombre = p.nombre;
        this.edad = p.edad;
    }
}

File: Taller Tipos de clases/Go/Clase Normal/Persona.go
class Persona
{
    public string Nombre;
    public int Edad;

    public Persona()
    {
        Nombre = "Sin nombre";
        Edad = 0;
    }

    public Persona(string nombre, int edad)
    {
        Nombre = nombre;
        Edad = edad;
    }

    public Persona(string nombre)
    {
        Nombre = nombre;
        Edad = 18;
    }

    public Persona(Persona p)
    {
        Nombre = p.Nombre;
        Edad = p.Edad;
    }
}

